### DISCUSSION TITLE
MacvlanEndpoint does not support Hot attach: unknow

### DISCUSSION THREAD

**Discussion by MJmichael** [kata-containers/kata-containers#7622] [Q&A]
Hi, everyone:
docker network create -d macvlan --subnet 10.0.105.0/24 --gateway 10.0.105.1 mynet
docker run -itd --network=mynet --ip 10.0.105.100 centos /bin/bash

I got "failed to create task for container: failed to create shim task: MacvlanEndpoint does not support Hot attach: unknown" ？ Does kata-runtime V2 suppport MacVlan？

**Comment by yarona1993** [comment]
Is there any update on this issue? also failed with IPVlanEndpoint
